10주차/BOJ_1100_하얀칸_저문.py

```diff
@@ -0,0 +1,18 @@
+chess = []
+for _ in range(8):
+    chess.append(input())
+
+temp = 0
+count = 0
+for i in range(8):
+    for j in range(8):
+        if i % 2 == 0:
+            if temp % 2 == 0 and chess[i][j] == 'F':
+                count += 1
+            temp += 1
+        else:
+            if temp % 2 == 1 and chess[i][j] == 'F':
+                count += 1
+            temp += 1
```

Comment:
i + j 가 홀수인지 짝수인지로 구분할 수 있을 것 같아요

```suggestion
            if (i+j) % 2 == 0 and chess[i][j] == 'F':
                count += 1
```

Reply:
천재..?